ui/core/components/settings_menu.tsx

```diff
@@ -152,5 +160,31 @@ export class SettingsMenu extends BaseModal {
 					sim.setShowQuickSwap(eventID, newValue);
 				},
 			});
+
+		if (useConcurrentWorkersWrap.value && useConcurrentWorkers.value) {
+			const values: EnumValueConfig[] = [{value: 0, name: "Off"}];
+			for (let i = 2; i < navigator.hardwareConcurrency + 1; i++) {
```

Comment:
This logic can be reduced and returned into a array like this:
```
const values = navigator.hardwareConcurrency.reduce<EnumValueConfig[]>(
  (acc, _, i) => {
    acc.push({ value: i, name: i.toString() });
    return acc;
  },
  [{ value: 0, name: "Off" }]
);
```

Reply:
navigator.hardwareConcurrency is a number value. How is that supposed to work?
Also I explicitly omit 1 because it doesn't make sense in this context, so a good old boomer loop seems to be the easiest solution :D

Reply:
Ah yea, should've added:  `Array(navigator.hardwareConcurrency)` I blindly wrote it